### PR TITLE
update cardinal care rules and ui

### PR DESCRIPTION
**Pull request by labhinav**

### CODE REVIEW

**Review by pjames27 — CHANGES_REQUESTED** [review]
There are some things that I'm happy with being on your dev branch, but would prefer not go into main. In my requested changes below, when I mention "don't push" something, feel free to keep them on your dev branch.
Going forward, I'm happy to review your PR's more frequently to give you a better idea of what I'm expecting. (E.g. I'll be somewhat particular about making changes to files everyone is using, such as epilog.js and load_common_data.js.)

If you have any questions, do let me know.

Requesting the following changes:
- Don't push the .ds_store files
- Don't push the changes to chubb_hospital_cash.js
- Update index.js to generate a link to your cardinal care page
- See comment in "cardinal_care.html"
- Don't push changes to epilog.js, even if it's just encoding or formatting
- See comment on load_common_data.js
- Please rename "load_example_claim.js" or move to another folder (e.g. "src/{abhinav,harshit}dev/", "src/{abhinav,harshit}-utils/", or "src/dev-utils/"), or leave on dev branch. I think the file is useful for development and you should use it, but it shouldn't be in src for general use unless it's either more generalized, or it's used in our public-facing version of the site.

File: src/load_common_data.js [hunk 1]
```diff
@@ -392,4 +392,6 @@ country.continent(vietnam,asia)
 country.continent(western_samoa,asia)
 country.continent(yemen,asia)
 country.continent(yugoslavia,europe)
-`;
\ No newline at end of file
+`;
+
+module.exports = {load_common_data};
```
[the remark this answers is not in the document]
Comment (labhinav) [reply src/load_common_data.js:397]: Acknowledged.

File: claims_analysis/cardinal_care/cardinal_care.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Cardinal Care Dev - Cardinal Care</title>
+  <link rel="stylesheet" href="cardinal_care.css">
+</head>
+<body>
+  <nav>
+    <a href="../../index.html">Cardinal Care</a>
```
[the remark this answers is not in the document]
Comment (labhinav) [reply claims_analysis/cardinal_care/cardinal_care.html:10]: Fixed

**Comment by labhinav** [comment]
Thanks for the comments, I've resolved them . Will pay more attention on what goes into PRs going forward

**Pull request merged**